buildtools/maven/pom_test.go

```diff
@@ -36,15 +38,25 @@ func TestPomFileGraph(t *testing.T) {
 	assert.Equal(t, wantTransitive, got.Transitive)
 }
 
+func TestPomFileDependencyManagement(t *testing.T) {
+	got, err := maven.PomFileGraph("dependencyManagement.xml", testPoms)
+	assert.NoError(t, err)
+	assert.Len(t, got.Direct, 1)
+	helpers.AssertPackageImport(t, got.Direct, "junit:junit", "3.8.2")
+
+	assert.Len(t, got.Transitive, 1)
+	packageOne := helpers.PackageInTransitiveGraph(got.Transitive, "junit:junit", "3.8.2")
+	assert.Len(t, packageOne.Imports, 0)
```

Comment:
this will always pass, even if the package isn't found, won't it?

Reply:
Yea, I need to add `assert.NotEmpty(t, packageOne)` here. I'm leaving the length check because I want to ensure that no transitive deps are created.